abuseipdb/src/abuseipdb.py

```diff
@@ -99,33 +100,47 @@ def _process_message(self, data):
             )
             return "IP found in AbuseIPDB WHITELIST."
         if len(data["reports"]) > 0:
+            found = []
+            cl = defaultdict(dict)
+
             for report in data["reports"]:
+                countryN = report["reporterCountryCode"]
+                if countryN in cl:
+                    cl[countryN]["count"] += 1
+                    cl[countryN]["firstseen"] = report["reportedAt"]
+                else:
+                    cl[countryN]["count"] = 1
+                    cl[countryN]["firstseen"] = report["reportedAt"]
+                    cl[countryN]["lastseen"] = report["reportedAt"]
```

Comment:
Just curious, are the reports sorted in a way, so that the new `first_seen` is always smaller than the old one?

Reply:
Ok great, thanks for the feedback! :)